### ISSUE TITLE
Use with VRTK

### ISSUE THREAD

**Issue by timmythetiny** [raphael-ernaelsten/Aura#35]
Forgive me, this isn't really an issue.  I was hoping to share my solution to using this tool with Steam VR, and I wasn't sure of a better place to do it.  The problem is that the VRTK manager doesn't make the camera with the Aura component available until after the volumes and lights have already had OnEnabled called, so they never get registered with the system.  My solution was to start a coroutine that enables all the volumes and lights once an Aura instance becomes available, as below:
`private WaitForEndOfFrame eof = new WaitForEndOfFrame();

void Start()
{
    StartCoroutine(LateEnable());
}

IEnumerator LateEnable() {
    while (Aura.Instance == null)
    {
        yield return eof;
    }

    AuraVolume[] volumes = FindObjectsOfType<AuraVolume>();
    AuraLight[] lights = FindObjectsOfType<AuraLight>();

    foreach (AuraVolume volume in volumes)
    {
        volume.enabled = true;
    }

    foreach (AuraLight light in lights)
    {
        light.enabled = true;
    }
}`
It's a simple fix, but this took quite a bit of digging for me to solve, so hopefully this will be of use to someone.  I can move this to the Wiki if one becomes available.

**Comment by nbac** [comment]
interesting that you use it with vrtk since i cant use it with vr at all . the noise is moving with my view matrix what am i doing wrong than?

**Comment by raphael-ernaelsten** [comment]
Thank you for sharing this!

**Comment by huzhangyang** [comment]
@nbac  are you using single-pass? multi-pass works fine for me.

**Comment by timmythetiny** [comment]
@nbac I haven't had that issue, no.

**Comment by nbac** [comment]
multipass since there is no support for singlepass yet. strange. i have to try it again because it looked really good in vr but i did switch back to true volumetric lights solution because of this. which unity version do you use and does steam vr mean you use it with vive? how is performance anyway for you for me it does cost 20 - 30 fps !?

**Comment by timmythetiny** [comment]
Right, we use it with Vive.  It can really tank performance if you have too many lights going on, for sure.  We're on 2018.1.  By "true" volumetric lights, do you mean Unity's solution from Adam?
